frontend/src/metabase/dashboard/containers/DashboardApp.jsx

```diff
@@ -124,6 +124,12 @@ const DashboardApp = props => {
 
   useOnUnmount(props.reset);
 
+  useEffect(() => {
+    const hash = parseHashOptions(props.location.hash);
+    setEditingOnLoad(hash.edit);
+    setAddCardOnLoad(hash.add && parseInt(hash.add));
+  }, [props.location.hash]);
```

Comment:
Seems like we already have parsed options there, wdyt about this:
```suggestion
  useEffect(() => {
    setEditingOnLoad(options.edit);
    setAddCardOnLoad(options.add && parseInt(options.add));
  }, [options.add, options.edit]);
```